tests/PHPStan/Command/ErrorFormatter/BaselineNeonErrorFormatterIntegrationTest.php

```diff
@@ -15,7 +15,7 @@ public function testErrorWithTrait(): void
 	{
 		$output = $this->runPhpStan(null);
 		$errors = Json::decode($output, Json::FORCE_ARRAY);
-		$this->assertSame(4, array_sum($errors['totals']));
```

Comment:
I guess we need to change this assertion because of the newly added testfiles under data/